patients/repo.go

```diff
@@ -1367,40 +1585,44 @@ func (r *repository) TideReport(ctx context.Context, clinicId string, params Tid
 		Config: TideConfig{
 			ClinicId: clinicId,
 			Filters: TideFilters{
-				TimeInVeryLowPercent:      ">0.01",
-				TimeInAnyLowPercent:       ">0.04",
-				DropInTimeInTargetPercent: "<-0.15",
-				TimeInTargetPercent:       "<0.7",
-				TimeCGMUsePercent:         "<0.7",
+				TimeInVeryLowPercent:      strp(">0.01"),
```

Comment:
Is this because these are no longer marked as required in the API spec? Consider if you might want to use `x-go-type-skip-optional-pointer` in TidepoolApi to keep these as plain strings so we don't have more pointer soup?

Or is it `x-omitzero` now?

https://github.com/oapi-codegen/oapi-codegen/discussions/2027

Reply:
Correct.